GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/DataTypes/TPRBlockGrid.config

```diff
@@ -1248,6 +1248,27 @@
       "stylesheet": "~/css/govuk-umbraco-backoffice.css",
       "thumbnail": null,
       "view": "~/App_Plugins/ThePensionsRegulator.Frontend.Umbraco/blocks/views/TPRBox.html"
+    },
+    {
+      "allowAtRoot": true,
+      "allowInAreas": true,
+      "areaGridColumns": null,
+      "areas": [],
+      "backgroundColor": null,
+      "columnSpanOptions": [],
+      "contentElementTypeKey": "5d33cd38-b819-4afd-9d4a-499ee73799ab",
+      "editorSize": "medium",
+      "forceHideContentEditorInOverlay": false,
+      "groupKey": "fe4459ab-dada-4c2f-97f2-1b70aee27310",
+      "iconColor": null,
+      "inlineEditing": false,
+      "label": "",
```

Comment:
In each example where you add this to the configuration for a block list or block grid data type you could add the following label: `Section break ({{$settings.size}}{{$settings.hidden ==  1 ? ', hidden' : ''}})`.

![image](https://github.com/user-attachments/assets/003e764c-6cf8-46f1-9368-127b10f086e8)

This means that the backoffice view will change from this:

![image](https://github.com/user-attachments/assets/9d5d900d-d1dd-4088-bbdb-12494eb95df0)

To this, which is more helpful:

![image](https://github.com/user-attachments/assets/ab3602c6-af2b-4258-9c85-fa8dc8c7a22e)